internal/node_client/config/bitcoin.conf

```diff
@@ -0,0 +1,145 @@
+server=1
```

Comment:
Why do we need this file here?

Reply:
@arkadiuszos4chain This files is used mounted in the docker container in the integration tests here
https://github.com/bitcoin-sv/arc/blob/088e810065b64681fbdda3416be6f3eb0747293f/internal/test_utils/docker.go#L136